ui/src/components/Organization/List.tsx

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames/bind';
+import style from 'src/components/Organization/style.module.scss';
+
+const c = classNames.bind(style);
+
+interface Organizations {
+  id: string;
+  name: string;
+  banner?: string;
+  square_logo?: string;
+}
+
+interface FieldProps {
+  list: Organizations[];
+}
+
+const List: React.FC<FieldProps> = ({ ...p }) => {
+  const organizations = p.list.map((e) => (
+    <div className={c('list-content')} key={e.id}>
+      <div className={c('list-section1')}>
```

Comment:
This class is named `list-section1`, but this is is the only section. What was the need to wrap the content of a list item in 2 `div`s?